docs/articles/documentation/test-api/actions/upload.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: docs
+title: Upload
+permalink: /documentation/test-api/actions/upload.html
+---
+# Upload
+
+Use the following actions to work with file upload input elements.
+
+* [Populate File Upload Input](#populate-file-upload-input)
+* [Clear File Upload Input](#clear-file-upload-input)
+
+## Populate File Upload Input
+
+Populates the specified file upload input with file paths.
+
+```text
+setFilesToUpload( selector, filePath )
+setFilesToUpload( selector, [filePath, ...] )
+```
+
+Parameter  | Type                                              | Description
+---------- | ------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------
+`selector` | Function &#124; String &#124; Selector &#124; Snapshot &#124; Promise | Identifies the input field to which file paths are written. See [Selecting Target Elements](index.md#selecting-target-elements).  
+`filePath` | String                                            | The path to the uploaded file. Relative paths resolve from the folder with the test file.
```

Comment:
Honestly, the part after `selector` in `t.setFilesToUpload( selector, [filePath, ...] )` looks more like an optional variadic arguments(e.g. `setFilesToUpload(selector)`, `setFilesToUpload(selector, path1, path2, path3)`), Maybe it's better to declare it as ``t.setFilesToUpload( selector, filePaths)` and describe `filePaths` like `filePaths | Array | ...` ?